extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzScheduler.java

```diff
@@ -89,6 +98,19 @@ public QuartzScheduler(SchedulerContext context, QuartzSupport quartzSupport, Co
                 CronDefinition def = CronDefinitionBuilder.instanceDefinitionFor(cronType);
                 CronParser parser = new CronParser(def);
 
+                InstanceHandle<UserTransaction> handle = Arc.container().instance(UserTransaction.class);
+
+                boolean nonManagedTx = quartzSupport.getBuildTimeConfig().getStoreClass()
+                        .equals("org.quartz.impl.jdbcjobstore.JobStoreCMT");
+
+                if (nonManagedTx) {
+                    if (handle.isAvailable()) {
+                        handle.get().begin();
+                    } else {
+                        LOGGER.debugf("Unable to start transaction for JobStoreCMT job store class. " +
```

Comment:
I think that this should be an error, or?

Reply:
I think we can as well remove this message since, transaction support will be brought transitively by the agroal extension. In build time we have a check that the datasource is properly configured. 

WDYT?